src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1949,6 +1949,12 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
         [Fact]
         public async Task SendAsync_RequestVersionNotSpecified_ServerReceivesVersion11Request()
         {
+            // Managed handler treats 0.0 as a bad version, and throws.
+            if (UseManagedHandler)
+            {
+                return;
+            }
```

Comment:
It looks like .NET Framework also fails on 0,0?  It'd be good to validate that we throw with the same exception.

But for the purposes of this test, do we actually need to use 0,0?  The comment below suggests that we just need it to be something "unknown".  What happens if we instead use 1,9 or something like that?